??

```diff
@@ -0,0 +1,9 @@
+let arr3 = [6, 2, 12, 14, -24, 5, 0];
```

Comment:
Arrays actually should be "const" instead of let. You can still manipulate the contents of the array when it's a constant and helps limit surface area for bugs.